packages/core/lib/testing/SolidityTest.js

```diff
@@ -3,7 +3,7 @@ const Suite = require("mocha/lib/suite.js");
 const Deployer = require("@truffle/deployer");
 const { Compile } = require("@truffle/compile-solidity");
 const { Shims } = require("@truffle/compile-common");
-const RangeUtils = require("@truffle/compile-solidity/compilerSupplier/rangeUtils");
+const RangeUtils = require("@truffle/compile-solidity/dist/compilerSupplier/rangeUtils");
```

Comment:
Geez, I hope that goes away in an upcoming PR... maybe mark this?

Reply:
It doesn't go away.

I don't have a good answer to this: in db and codec, etc., the approach we've settled on is to export everything in one module, subdivided into namespaces. Unfortunately, this means you can't do any tree-shaking; all the modules everywhere end up imported all at once, resulting in a large module size in memory and risk of incompatibility with things like `fs`.

I've left these here for now because I didn't want to go ahead and follow this prior approach here, knowing now the pretty severe downsides. My conclusion was that this deep-package import could really be considered internal to Truffle (users of the package very likely won't need to do this), and so it seemed acceptable for the time being.

Would love to discuss this though.

Reply:
Blech. :-/  Well, if you think this is the approach to take, I'll leave it to you.